content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,203 @@
+
+.. _barcode/operations/scan-received-products:
+
+Scan barcodes for receipts
+==========================
+
+To process warehouse receipts for incoming products, there first needs to be a purchase order (PO)
+created, and a receipt operation to process.
+
+To create a :abbr:`PO (purchase order)`, navigate to the :menuselection:`Purchase app --> Create` to
+create a new request for quotation (RFQ).
+
+From the blank :abbr:`RFQ (request for quotation)` form, click the drop-down menu next to the
+:guilabel:`Vendor` field to add a vendor. Then, on the :guilabel:`Product` line under the
+:guilabel:`Products` tab, click :guilabel:`Add a product`, and select the desired product(s) to add
+to the quotation.
+
+Once ready, click :guilabel:`Save` at the top of the form, then click :guilabel:`Confirm Order` to
+confirm the :abbr:`RFQ (request for quotation)` to a :abbr:`PO (purchase order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-purchase-order.png
+   :align: center
+   :alt: Completed purchase order for barcode product.
+
+To process and scan barcodes for warehouse receipts, navigate to the :menuselection:`Barcode app`.
+
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process receipts, click on the :guilabel:`Operations` button at the bottom
+of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+.. image:: receipts_deliveries/receipts-deliveries-barcode-scanner.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+From this page, locate the :guilabel:`Receipts` card, and click the :guilabel:`# To Process` button
+to view all outstanding receipts. Then, select the desired receipt operation to process. This
+navigates to the barcode transfer screen.
+
+.. note::
+   If *only* using a barcode scanner or the Odoo mobile app, the barcodes for each transfer of a
+   corresponding operation type can be scanned to be processed easily. Once scanned, the products
+   that are part of an existing transfer can be scanned, and new products can be added to the
+   transfer, as well. Once all products have been scanned, validate the transfer to proceed with the
+   stock moves.
+
+From this screen, an overview of all receipts to process within that transfer (**WH/IN/000XX**) is
+shown. At the bottom of the screen, there are options to :guilabel:`Add Product` or
+:guilabel:`Validate`, depending on if products need to be added to the operation, or if the whole
+operation should be validated at once.
+
+.. image:: receipts_deliveries/receipts-deliveries-scanner-overview.png
+   :align: center
+   :alt: Overview of receipts in transfer to scan.
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+#`
+button (in this case, :guilabel:`+10`) can be clicked to indicate receipt of that product, or the
+:guilabel:`pencil icon` can be clicked to open a new screen to edit that product line.
```

Comment:
Icon formatting - pull "icon" out of the guilabel and either include the emoji with the descriptor in parenthesis or just the descriptor :)
```suggestion
:guilabel:`✏️ (pencil)` icon can be clicked to open a new screen to edit that product line.
```